src/client.js

```diff
@@ -224,7 +241,6 @@ export default class StitchClient {
    *
    * @returns {Promise}
    */
-
   doSessionPost() {
```

Comment:
I cannot comment on this since it's not in the change, but I don't think `userProfile` works anymore since the user profile is now at `auth/profile`.

Reply:
Is there a good way you want me to test this?  I can't find a test around this function anywhere...  Or should I just /shrug and delete it?

Reply:
Well we can't delete it. It's a needed method to get your profile. This is suitable as an integration test that you run after login. The profile contains details about your identities and such.